cmd/kube-version-change/version.go

```diff
@@ -78,7 +78,14 @@ func main() {
 	if err != nil {
 		log.Fatalf("Couldn't read from input: %q", err)
 	}
+	isYAML := isYAML(data)
 
+	if isYAML {
```

Comment:
Thanks.

Without a test, this is probably going to be broken again.

@lavalamp